src/main/docker/Dockerfile

```diff
@@ -1,13 +1,13 @@
-FROM fabric8/java-jboss-openjdk8-jdk:1.2
+FROM tomcat:8.0-jre8
 MAINTAINER Eiffel-Community
-USER root
-ENV JAVA_APP_DIR=/deployments
-ENV JAVA_APP_JAR=eiffel-intelligence-frontend.war
-EXPOSE 8092 8778 9779
 
-# Create image with existing war file. User need to execute 'mvn package -DskipTest' before 'docker build'
-COPY ./target/eiffel-intelligence-frontend-*.war /deployments/eiffel-intelligence-frontend.war
+ARG URL
+RUN echo Building Eiffel-Intelligence Front-end image based on artifact url: ${URL}
 
-CMD /deployments/run-java.sh
+# Create image with existing war file. User need to execute 'mvn package -DskipTest' before 'docker build'
+RUN ["rm", "-fr", "/usr/local/tomcat/webapps/ROOT"]
```

Comment:
Same question here as in eiffel-intelligence backend.

Reply:
This comes from early versions of @emichaf Dockerfiles,, don't know why we have it. Looks like to cleaning the default Tomcat webapp which is replaced by Eiffel-Intelligence Front-end app.
The difference of running the command a brackets "[]" and not, I think its something envrionment and java properties is treated differently. But in this case is should not matter which format that is used, since it is only a normal remove command that is executed.